script.js

```diff
@@ -1,4 +1,68 @@
+var userWins = 0;
+var computerWins = 0;
+
+var numberGenerator = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var finalNumber = randomInteger + 1;
+  return finalNumber;
+};
+
+var assignMoveToNumber = function () {
```

Comment:
Perhaps this function would be clearer named `generateRandomItem` or something like that, focusing on the outcome instead of what happens inside.